src/servers/authentication/src/email.mjs

```diff
@@ -22,25 +15,11 @@ export const sendEmail = async email => {
 		<a href="${link.toString()}">Click here</a><br>
 		If you don't verify your email, your token above will not work.`;
 
-	const input = {
-		Source: 'authenticate@dap-tools.uk',
-		Destination: {
-			ToAddresses: [email],
-		},
-		Message: {
-			Body: {
-				Html: {
-					Charset: 'UTF-8',
-					Data: message,
-				},
-			},
-			Subject: {
-				Charset: 'UTF-8',
-				Data: 'Nesta verification email',
-			},
-		},
-	};
-	const command = new SendEmailCommand(input);
-	const response = await client.send(command);
+	const response = await sendEmail(
+		email,
+		'authenticate@dap-tools.uk',
```

Comment:
I've seen this email in multiple points, I'd have it in a single config and import it, to ease future maintenance